common/v2/components/SignTransactionWallets/Web3.tsx

```diff
@@ -67,7 +67,7 @@ class SignTransactionWeb3 extends Component<ISignComponentProps & INetworkContex
     }
   }
 
-  public componentWillMount() {
+  public UNSAFE_componentWillMount() {
```

Comment:
does this change functionality at all?

Reply:
It's been deprecated so to avoid the warning in the console we need to prefix with `UNSAFE_`